etc/adminhtml/system.xml

```diff
@@ -197,6 +197,15 @@
                     </comment>
                     <source_model>Nosto\Tagging\Model\Config\Source\Memory</source_model>
                 </field>
+                <field id="category_sorting" translate="label comment" type="select" sortOrder="100"
+                       showInDefault="0" showInWebsite="0" showInStore="1">
+                    <label>Category sorting (experimental)</label>
+                    <comment>
+                        <![CDATA[Experimental feature. Might be removed in future releases. Contact Nosto for further details.]]>
+                    </comment>
+                    <source_model>Magento\Config\Model\Config\Source\Yesno</source_model>
```

Comment:
We must use a custom source model here and we need to indicate if the tokens are missing similar to M1. Currently the UI doesn't indicate in any way why the dropdown is disabled